pkg/util/validation/limits.go

```diff
@@ -214,6 +215,7 @@ func (l *Limits) RegisterFlags(f *flag.FlagSet) {
 	f.IntVar(&l.RulerTenantShardSize, "ruler.tenant-shard-size", 0, "The tenant's shard size when sharding is used by ruler. Value of 0 disables shuffle sharding for the tenant, and tenant rules will be sharded across all ruler replicas.")
 	f.IntVar(&l.RulerMaxRulesPerRuleGroup, "ruler.max-rules-per-rule-group", 20, "Maximum number of rules per rule group per-tenant. 0 to disable.")
 	f.IntVar(&l.RulerMaxRuleGroupsPerTenant, "ruler.max-rule-groups-per-tenant", 70, "Maximum number of rule groups per-tenant. 0 to disable.")
+	f.IntVar(&l.RulerMaxOutputsPerRule, "ruler.max-outputs-per-rule", 10000, "Maximum number of alerts/series each rule can generate. 0 to disable.")
```

Comment:
"outputs" isn't super clear to me. WDYT about renaming it to `RulerMaxSeriesPerRule`?

Reply:
@dimitarvdimitrov is `Series` the right term here, considering the limit controls both the number of alerts _and_ series a rule can generate?

Reply:
each alert is generated from a series that the query returns, so to me they are the same. Maybe "results" is a better term?

Reply:
@dimitarvdimitrov I guess results would be a descriptive term.

Reply:
@dimitarvdimitrov done.